snooty/parser.py

```diff
@@ -107,6 +109,12 @@ class ProjectLoadError(Exception):
     pass
 
 
+class ChildValidationError(Exception):
+    """Empty exception expected to continue"""
```

Comment:
For me... what does this mean? Not sure if this isn't clear or if I'm misunderstanding.

Reply:
Nah, you're right. It can be quite unclear without context. I tried to convey that this exception intentionally doesn't do much given that it's only supposed to be used to help call "continue" in some loops used for validation when it's raised.

I went ahead and just removed the docstring because it sounds unnecessarily and confusingly prescriptive. The class name and current usage are probably good enough to give an idea of what it's for. Wdyt?

Reply:
I'm fine with this, thanks!